contracts/modules/Payments.sol

```diff
@@ -137,4 +138,37 @@ abstract contract Payments is RouterImmutables {
             }
         }
     }
+
+    /// @notice Wrap an amount of stETH into wstETH
+    /// @param recipient The recipient of the stETH
+    /// @param amount The amount of wstETH desired
+    function wrapSTETH(address recipient, uint256 amount) internal {
+        if (amount == Constants.CONTRACT_BALANCE) {
+            amount = STETH.balanceOf(address(this));
+        } else if (amount > STETH.balanceOf(address(this))) {
+            revert InsufficientSTETH();
+        }
+        if (amount > 0) {
+            WSTETH.wrap(amount);
+            if (recipient != address(this)) {
+                WSTETH.transfer(recipient, amount);
```

Comment:
wouldn't the initial amount of STETH be greater than the amount of WSTETH we get back from wrapping and thus we won't have enough to transfer here?